astropy_iers_data/__init__.py

```diff
@@ -14,7 +14,7 @@
     "IERS_B_README",
     "IERS_LEAP_SECOND_FILE",
     "IERS_LEAP_SECOND_URL",
-    "IETF_LEAP_SECOND_URL"
```

Comment:
Is IETF a typo?

Reply:
No it seems it was deliberate before otherwise it would be the same as the variable above. I think it's clearer with this PR though!